scripts/index.js

```diff
@@ -1,9 +1,11 @@
 const rssFeed = require('./build-rss');
 const buildRoadMap = require('./build-roadmap');
 const buildPostList = require('./build-post-list');
+const buildIssues = require('./build-issues');
 
 async function start() {
   buildRoadMap();
+  buildIssues();
```

Comment:
I think we should build it like we do it with meetings.json, on daily basis, to have latest data. 
If we keep it at build level, we depend on website build which sometimes can happen few times a day, sometimes once a week

references:
- https://github.com/asyncapi/website/blob/master/package.json#L13
- https://github.com/asyncapi/website/blob/master/.github/workflows/regenerate-meetings-and-videos.yml